connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockMoveEntityAbsoluteTranslator.java

```diff
@@ -39,8 +40,14 @@
 
     @Override
     public void translate(MoveEntityAbsolutePacket packet, GeyserSession session) {
+        float y = packet.getPosition().getY();
+        if (session.getRidingVehicleEntity() instanceof BoatEntity) {
+            // Remove some Y position to prevents boats from looking like they're floating in water
+            // Not by the boat offset because 1.16.100 complains and that's probably not good for the future
+            y -= 0.3f;
```

Comment:
Where is this `-0.3` pulled from? Can it be stored as a constant somewhere?

Reply:
It's something close to the boat entity's offset but not exactly (which messes up 1.16.100 as stated). I can make it a constant.